pyproject.toml

```diff
@@ -92,6 +98,8 @@ disable = """,
     too-few-public-methods,
     duplicate-code,
     too-many-ancestors,
+    unspecified-encoding,
+    consider-using-f-string,
```

Comment:
It's a step backwards IMHO for us to add new pylint global disables. Better to fix these or disable them case-by-case if absolutely necessary.

Reply:
sure, I am just trying one to make it fail due mysql, and then I will refine the whole PR